### PR TITLE
Export enter and continue MPIPL chip-ops

### PR DESCRIPTION
**Pull request by dhruvibm**

### CODE REVIEW

**Comment by amitay** [comment]
I would prefer the public api for mpipl to have sbe_ prefix. These hardware procedures are likely to be only implemented via sbe.

**Comment by amitay** [comment]
Also, the api will expect a pib target and not sbe target.  Please update documentation accordingly.

File: libpdbg/hwunit.h [hunk 1]
```diff
@@ -69,6 +69,8 @@ struct chipop {
 	struct pdbg_target target;
 	uint32_t (*ffdc_get)(struct chipop *, const uint8_t **, uint32_t *);
 	int (*istep)(struct chipop *, uint32_t major, uint32_t minor);
+        int (*mpipl_enter)(struct chipop *);
```
Comment (devenrao) [inline libpdbg/hwunit.h]: most of the pdbg code tab is enabled, so do insert tabs instead of spaces

File: libpdbg/sbefifo.c [hunk 2]
```diff
@@ -179,6 +179,22 @@ static int sbefifo_op_istep(struct chipop *chipop,
...
+}
+
+static int sbefifo_op_mpipl_enter(struct chipop *chipop)
+{
+        struct sbefifo *sbefifo = target_to_sbefifo(chipop->target.parent);
+        struct sbefifo_context *sctx = sbefifo->get_sbefifo_context(sbefifo);
+
+        return sbefifo_mpipl_enter(sctx);
+}
+
```
Comment (devenrao) [inline libpdbg/sbefifo.c]: inser tabs

**Comment by amitay** [comment]
Please split the patch into two. One to add hw procedure to sbefifo and the second one which adds public api in libpdbg.  Also, in documentation you need to define what the return value is.

**Comment by dhruvibm** [comment]
Hi made two commits based on the comment and added the return documentation.

**Pull request closed without merging**